### DISCUSSION TITLE
Configure JWT header key

### DISCUSSION THREAD

**Discussion by rianfowler** [gravitational/teleport#10539] [Q&A] [answered]
Regarding [headers passthrough](https://goteleport.com/docs/application-access/guides/connecting-apps/#headers-passthrough): Is there a way to configure the Teleport cluster proxy to pass the JWT with a different header key than `Cf-Access-Token` or `Teleport-Jwt-Assertion`? [This](https://goteleport.com/docs/application-access/guides/jwt/#payload) suggest that there isn't a way to do this. 

I've tried a few guesses for trying to template it into the `apps[].rewrite.headers:` config ([similar](https://goteleport.com/docs/application-access/reference/#application-resource) to how `external.env` is templated into the `X-External-Trail:` header). None of them have worked.

Thanks!

**Comment by webvictim** [comment, marked as answer]
Outside of recompiling, there isn't currently a way to configure Teleport to send JWT assertions using a different header. It's often easier to configure the target application to look for the value in a different header.
